model/prelude_mem.sail

```diff
@@ -60,13 +60,13 @@ struct RISCV_strong_access = {
    with additional information, provided we can supply a function that
    converts it into a bitvector.  Since we are just using xlenbits as a
    physical address, we need the identity function for xlenbits. */
-val xlenbits_identity : xlenbits -> xlenbits
+val physaddrbits_identity : physaddrbits -> bits(64)
 
-function xlenbits_identity xs = xs
+function physaddrbits_identity xs = zero_extend(xs)
```

Comment:
Maybe this should be called `_zero_extend` instead of `_identity`?

Reply:
Sounds great ! Done